docs/web3.eth.rst

```diff
@@ -654,6 +654,19 @@ The following methods are available on the ``web3.eth`` namespace.
         '0x1a8bbe6eab8c72a219385681efefe565afd3accee35f516f8edf5ae82208fbd45a58f9f9116d8d88ba40fcd29076d6eada7027a3b412a9db55a0164547810cc401'
 
 
+.. py:method:: Eth.signTypedData(account, jsonMessage)
+
+    * Delegates to ``eth_signTypedData`` RPC Method
+
+    Please note that the ``jsonMessage`` argument is the loaded JSON Object
+    and **NOT** the JSON String itself.
```

Comment:
Not a blocker but it does seem somewhat trivial for us to support it in both formats.  Can be done as a follow up.  Easier to add support than remove it.

Reply:
Added issue #1342 to track!